egui_glow/src/misc_util.rs

```diff
@@ -0,0 +1,129 @@
+#![allow(unsafe_code)]
+use glow::HasContext;
+#[cfg(target_arch = "wasm32")]
+use wasm_bindgen::JsValue;
+
+pub(crate) fn srgbtexture2d(
+    gl: &glow::Context,
+    compatibility_mode: bool,
```

Comment:
this needs a docstring (compatibility with what?)

Reply:
Maybe `is_webgl_1` is a better name?

Reply:
Ok i will fix it.